include/flatbuffers/flatbuffers.h

```diff
@@ -599,6 +596,31 @@ class vector_downward {
     scratch_ = buf_;
   }
 
+  void clear_allocator()
+  {
```

Comment:
`{` on prev line, same below

Reply:
thx, I couldnt somehow use clang-format -> kind of messes up everything, when I apply it... (i use clang 6.0 maybe thats why...